openssl/openssl.go

```diff
@@ -160,7 +160,7 @@ func GetECPointFromAffine(affineX, affineY BigNum, curve Curve) (ECPoint, error)
     return newPoint, nil
 }
 
-func GetAffineCoordsFromECPoint(point ECPoint, curve Curve) (BigNum, BigNum, error) {
+func GetAffineCoordsFromECPoint(point ECPoint, curve *Curve) (BigNum, BigNum, error) {
     // affineX and affineY must be freed later by the calling function.
```

Comment:
can you just take curve from the point?

Reply:
Since `goUmbral/math` imports `goUmbral/openssl`, `goUmbral/openssl` won't be able to use Point due to a circular dependency. You make a good point that this function basically takes a Point struct though.